ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/writers/TomlComposer.kt

```diff
@@ -0,0 +1,54 @@
+package com.akuleshov7.ktoml.writers
+
+import com.akuleshov7.ktoml.writers.IntegerRepresentation.DECIMAL
+
+/**
+ * Abstracts the specifics of writing TOML files into "emit" operations.
+ */
+public interface TomlComposer {
```

Comment:
No specific plans for it, I just thought this way was best practice. I can merge it if that's preferable

Reply:
Yes, it is a best practice when it has common sense. But here we interface it "just because" 😄 
Better to merge, so the project won't be too huge to read for new developers